.github/ISSUE_TEMPLATE/system-tests.md

```diff
@@ -18,6 +18,73 @@ The following issue aims to run all `system tests` for the current release candi
 | **Tag**                              |                                            |
 | **Previous system tests issue**      |                                            |
 
+## Instructions
+For running tests in an AWS EC2 virtual environment, it will be needed to meet the following requirements:
+
+| Environment                  | EC2                                       |
+|------------------------------|-------------------------------------------|
+|Basic_cluster                 |Ubuntu 22.04.2 LTS `C5`.`XLarge` `15GB` HD |
+|Big_cluster_40_agents         |Ubuntu 22.04.2 LTS T3.Large `60GB` HD      |
+|Agentless_cluster             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Four_manager_disconnected_node|Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|One_manager_agent             |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Manager_agent                 |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+|Enrollment_cluster            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        | 
+| Basic_environment            |Ubuntu 22.04.2 LTS T3.Large 30GB HD        |
+
+
+These requirements should be requested from the @cicd-team.
+
+
+For its execution, the installation of various packages is required, which will be detailed below.
+
+<details><summary>Steps</summary>
+
+```
+### Updating dependencies
+sudo apt update
+sudo apt upgrade
+
+### Installing git and cloning wazuh-qa project
+sudo apt install git
+git clone https://github.com/wazuh/wazuh-qa.git
+
+### Installing docker
+sudo apt install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+echo "deb [signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt update
+sudo apt install -y docker-ce docker-ce-cli containerd.io
+sudo systemctl start docker
+sudo systemctl enable docker
+
+### Installing ansible
+sudo apt install ansible
+
+### Installing ansible-docker libraries
+sudo pip3 install docker
+sudo apt install python3-pip
+pip install docker-py
+
+### Restarting docker
+sudo systemctl restart docker
+sudo docker system prune -a -f
+
+### Access as a root
+sudo su
+
+### Installing requirements and additional libraries
+python3 -m pip install -r /home/ubuntu/wazuh-qa/requirements.txt
+sudo apt install python3-pip
+sudo apt install python3.10-venv
+cd /home/ubuntu/wazuh-qa/deps/wazuh_testing
+python3 -m pip install .
+```
+
+For further information: check /wazuh-qa/tests/system/README.md
```

Comment:
Set a complete URL

Reply:
Done in https://github.com/wazuh/wazuh/pull/19080/commits/c81065dc33d44b5ff2af146a3783d78205b70aa5